docs/index.md

```diff
@@ -0,0 +1,22 @@
+# Perseo Context-Aware CEP
+
+Perseo is an [Esper-based](http://www.espertech.com/esper/) Complex Event Processing (CEP) software designed to be fully
+_NGSIv2_-compliant. It uses NGSIv2 as the communication protocol for events, and thus, Perseo is able to seamless and
+jointly work with _context brokers_ such as [Orion Context Broker](https://github.com/telefonicaid/fiware-orion).
```

Comment:
This sentence should be changed in the same way the one in the main README.md

(This is a problem with rendundant text... a change in one place is forgotten in the other :) It would be great if Markdown could have some kind of "include" macro, but I don't know...)

Reply:
Fixed in https://github.com/telefonicaid/perseo-fe/pull/592/commits/ce3e52c048d89a72045670ba717cf35c190fa4e3

Now all the 3 files have the same approach